src/utils/require-template.js

```diff
@@ -17,7 +17,14 @@ function isValidTemplate(template) {
   );
 }
 
-module.exports = function(template) {
+const getOcTemplate = (path) => {
+  if (require.cache && !!require.cache[path]) {
+    delete require.cache[path];
+  }
+  return require(path);
+};
+
+module.exports = function (template) {
   let ocTemplate;
   const localTemplate = path.join(
```

Comment:
I think we need to understand, in case of yarn install, why this doesn't work. The oc-template-jade module must be there if yarn installed everything correctly, if it is not we need to find out where it is